docs/make.jl

```diff
@@ -25,7 +25,7 @@ makedocs(
 )
 
 deploydocs(
-    repo = "github.com/claresinger/Insolation.jl.git",
+    repo = "github.com/clima/Insolation.jl.git",
```

Comment:
```suggestion
    repo = "github.com/CliMA/Insolation.jl.git",
```

Reply:
I think we just need this fix, plus adding ` - master` [here](https://github.com/CliMA/Insolation.jl/blob/cs/fix-docs/.travis.yml#L25). We can check to see if the deployment is successful once it merges (we'll only know once this lands on the master branch).